types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -12,7 +12,7 @@ export enum BaseChatItemTypeProps {
 }
 
 export interface BaseChatItemProps {
-  avatar: string;
+  avatar: React.ReactElement;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `string` to `React.ReactElement` for the `avatar` prop in `BaseChatItemProps` interface might cause issues if there are any components that still pass a string URL as the avatar. Please ensure all components using this interface have been updated accordingly.

```diff
-  avatar: string;
+  avatar: React.ReactElement;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->